src/main/java/io/github/vhoyon/vramework/utilities/TimerManager.java

```diff
@@ -0,0 +1,42 @@
+package io.github.vhoyon.vramework.utilities;
+
+import java.util.HashMap;
+import java.util.Timer;
+import java.util.TimerTask;
+
+public final class TimerManager {
+
+	private static HashMap<String,Timer> timers;
+
+	private TimerManager(){}
+
+	/**
+	 * @param timerName Name of the timer
+	 * @param delay Delay (in milliseconds) before doing the action
+	 * @param action Action to do after the delay was finished, without being called again, which resets the delay
+	 */
+	public static void manage(final String timerName, final int delay, TimerTask action){

```

Comment:
```suggestion
	public static void manage(final String timerName, final int delay, Runnable action){
```

I would make that change so using TimerManager is as simple as possible, and the `TimerTask` object would be created inside here automatically.

Reply:
I am not sure that would actually work but i'll check it out

Reply:
Just doing that change wouldn't work indeed, but what I wanted to emphasize is the parameters : the method should have a runnable, and inside, it creates a new `TimerTask` ("automatically") and runs `action.run()`.

I suggest to change the `TimerTask` parameter to a `Runnable`, obviously the inner-working logic will need to change too ;)

Reply:
Yeah i realised that after asking